ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml

```diff
@@ -0,0 +1,313 @@
+kind: Template
```

Comment:
This is not correct.  You should have been using cluster-launch-e2e which is already configured against AWS - anything "special" to AWS should have been in an if block, or better, placed within the entry point.

Please open a follow up and converge this - this file should not exist.

Reply:
This is probably a wrong title in this PR. Openshift ansible team needs a bare-metal-like tests, which use `prerequisites` and `deploy_cluster` playbooks with openshift-ansible-specific settings - e.g. Atomic host etc. This is why there is a new template - so that openshift-ansible changes won't affect new installer.

So in https://github.com/openshift/openshift-ansible/pull/9744 a new provisoner was written. So far it supports AWS only, but this might change in the future.